source/_integrations/google_cloud.markdown

```diff
@@ -130,6 +135,7 @@ tts:
     speed: 0.9
     pitch: -2.5
     gain: -5.0
+	text_type: ssml
```

Comment:
Tab characters are not permitted in YAML. Replace with 4 spaces, in case somebody tries to copy and paste.

Reply:
Thanks!